population/create_glpi_computer_redfish.py

```diff
@@ -619,6 +660,31 @@ def post_to_glpi(  # noqa: C901
         print(str(computer_response) + "\n")
         COMPUTER_ID = computer_response.json()["id"]
 
+    # Get Rack Location from Sunbird if relevant flags are provided
+    if all(
```

Comment:
So this is just to short circuit all sunbird logic if one of the args is not present, correct?

Reply:
Correct - I modified the comment slightly to be more clear.